pages/common/age.md

```diff
@@ -0,0 +1,28 @@
+# age
+
+> A simple, modern and secure file encryption tool.
+> More information: <https://age-encryption.org>.
+
+- Generate an encrypted file that can be decrypted with a passphrase:
+
+`age --passphrase --output {{path/to/encrypted_file}} {{path/to/unencrypted_file}}`
+
+- Generate a keypair, saving the private key and the public key to an unencrypted keyfile and printing the public key to stdout:
```

Comment:
```suggestion
- Generate a key pair, saving the private key to an unencrypted file and printing the public key to stdout:
```

keypair and keyfile are not really words, and while the public key does end up in a comment in the output, I think it's a little confusing to say they are both saved.

Reply:
Hey there. Thanks for the review. In other pages we use both `keypair` and `key pair`, so I would suggest here that we go with whichever way the official documentation for the command does it.

Reply:
Well, the official documentation doesn't specify anything AFAICT, but @FiloSottile is the main developer of Age, so I guess we can trust him in this regard?

Reply:
Yes, let's do that. It's also better to use real words whereever possible 👍🏻

Reply:
Oh, cool! I didn't realise. Sounds good then, let's do this.